tbconnect/tasks.py

```diff
@@ -153,10 +153,12 @@ def send_tbcheck_data_to_cci(data):
         if (
             response.status_code == 200
             and b'"Received Successfully"' == response.content
+            or response.status_code == 200
+            and b'"Received Sucessfully"' == response.content
         ):
             return "CCI data submitted successfully"
         response.raise_for_status()
-        return "CCI data Submission failed"
+        return "CCI data Submission failed {}".format(response.content)
```

Comment:
Looking at this now I think this should also raise an exception so that we see it in sentry